src/TrackGenerator.cpp

```diff
@@ -1702,51 +1781,67 @@ void TrackGenerator::correctFSRVolume(int fsr_id, FP_PRECISION fsr_volume) {
   log_printf(INFO, "Correcting FSR %d volume from %f to %f",
              fsr_id, curr_volume, fsr_volume);
 
-  int num_segments, azim_index;
-  double dx_eff, d_eff;
-  double volume, corr_factor;
-  segment* curr_segment;
-  segment* segments;
+#pragma omp parallel
```

Comment:
Are you using FSR volume correction? Have you tested out this new parallel implementation? This code isn't used in any of our tests or sample inputs, so I'm wary to accept these changes without either a) confirmation that it works as expected from a test problem and/or b) a new test in the test suite to demonstrate that it works.